docs/devGuide/contributing.md

```diff
@@ -0,0 +1,71 @@
+<frontmatter>
+  title: "Contributing"
+  layout: devGuide
+  pageNav: default
+</frontmatter>
+
+# Contributing to MarkBind 
+
+Thank you for your interest in contributing! We're glad you want to help. Read the sections below to learn about our code of conduct and how to get started contributing.
+
+## Code of conduct
+
+This project and everyone participating in it are governed by our [Code of Conduct]({{baseUrl}}/devGuide/contributing/code-of-conduct.html). By participating, you are expected to uphold this code. Please report unacceptable behavior to markbind@comp.nus.edu.sg.
+
+## Things to do before getting started
+
+Make sure you have the project set up and ready. We have guides detailing what we [expect our contributors to know]({{baseurl}}/devguide/devguide#requirement), our required [environment]({{baseurl}}/devGuide.html#environment) and our [development process]({{baseurl}}/devGuide.html#development-process).
+
+## How to contribute
+
+All contributions are welcome. You can contribute to MarkBind by:
+- Reporting bugs
+- Requesting features
+- Making changes (with a corresponding Pull Request), by:
+   - Updating documentation
+   - Fixing bugs
+   - Implementing features 
+
+### How to report a bug
+
+Report bugs in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding.
```

Comment:
repitition of `Following the guidelines`, could change the second one to `This`
`undestanding` -> `understanding`